File: src/game/tools/rpc/output/wrap/ItemDataWrap.cs
using System;
using Common.Net.Simple;

namespace Game 
{
	public class ItemDataWrap : IWrapper
	{	
		public int ItemID; // 道具ID
		public int Count; // 道具数量
		public long LastUpdateTime; // 最后一次更新时间
		
		public void Decode(Packet pck)
		{	
			ItemID = pck.GetInt(); 
			Count = pck.GetInt(); 
			LastUpdateTime = pck.GetLong(); 
		}
		
        public void Encode(Packet pck)
        {	
        	pck.PutInt(ItemID); 
        	pck.PutInt(Count); 
        	pck.PutLong(LastUpdateTime); 
        }
	}
}

File: src/game/tools/rpc/output/wrap/ItemDataWrap.ts
class ItemDataWrap extends Net.Simple.IWrapper {
	
	public ItemID:number; // 道具ID
	public Count:number; // 道具数量
	public LastUpdateTime:Long; // 最后一次更新时间
	public constructor() {
		super();
	}
	Decode(pck:Net.Simple.Packet):ItemDataWrap{
		
		this.ItemID = pck.GetInt(); 
		this.Count = pck.GetInt(); 
		this.LastUpdateTime = pck.GetLong(); 
		return this;
	}
    Encode(pck:Net.Simple.Packet){
		
    	pck.PutInt(this.ItemID); 
    	pck.PutInt(this.Count); 
    	pck.PutLong(this.LastUpdateTime); 
	}

}